src/components/ec-menu/ec-menu.story.js

```diff
@@ -39,10 +39,10 @@ stories
       },
     },
     template: `
-      <div style="margin: 0 auto;" :style="{ maxWidth: width + 'px' }">
+      <div class="tw-my-auto tw-mx-auto" :style="{ maxWidth: width + 'px' }">
```

Comment:
my is 0 no auto, continue having the same behavior?

Reply:
Yes it has the same behaviour, anyway I'll fix it